ffc-ahwr-event/protective-monitoring.js

```diff
@@ -0,0 +1,31 @@
+const { PublishEvent } = require('ffc-protective-monitoring')
+
+async function sendEvent (context, event) {
+  const raisedEvent = event.properties
+  const { sbi, ip } = raisedEvent
+  const cph = raisedEvent.cph.replace(/\//g, '')
+  const protectiveMonitoring = new PublishEvent(process.env.MONITORING_URL)
+
+  await protectiveMonitoring.sendEvent({
+    sessionid: raisedEvent.id.toString(),
+    datetime: createEventDate(),
+    version: '1.1',
+    application: process.env.MONITORING_APPLICATION,
+    component: 'Annual health and welfare review of livestock',
+    ip,
+    pmccode: process.env.PMC_CODE,
```

Comment:
This code will need to be allocated from the SOC team.

We will need to engage with the SOC team for testing to make sure the event are been picked up by Azure Sentinel.